src/test/java/com/rapid7/client/dcerpc/mslsad/objects/Test_LSAPRPolicyAccountDomInfo.java

```diff
@@ -181,6 +181,6 @@ public void test_toString() {
         sid.setIdentifierAuthority(new byte[]{1, 2, 3, 4, 5, 6});
         sid.setSubAuthority(new long[]{1, 2, 3});
         obj.setDomainSid(sid);
-        assertEquals(obj.toString(), "LSAPR_POLICY_ACCOUNT_DOM_INFO{DomainName:RPC_UNICODE_STRING{value:\"test 123\", nullTerminated:false}, DomainSid:RPC_SID{Revision:1, SubAuthorityCount:4, IdentifierAuthority:[1, 2, 3, 4, 5, 6], SubAuthority: [1, 2, 3]}}");
+        assertEquals(obj.toString(), "LSAPR_POLICY_ACCOUNT_DOM_INFO{DomainName:RPC_UNICODE_STRING{value:\"test 123\", nullTerminated:false}, DomainSid:S-1-0x01 02 03 04 05 06-1-2-3}");
```

Comment:
Should there be spaces between the digits? Can S-1-0x01 02 03 04 05 06-1-2-3 be parsed successfully using smbj's SID or CifsSecurityIdentifier?

Reply:
I was trying to find Microsoft's official implementation of the [ConvertSidtoStringSid](https://msdn.microsoft.com/en-us/library/windows/desktop/aa376399(v=vs.85).aspx) function, this is the closest I could find : http://www.pinvoke.net/default.aspx/advapi32.convertsidtostringsid . They don't seem to have spaces. I assumed we can remove this code if / when we extend smbj's SID.class

Reply:
You could check with win32security python package.

Reply:
Fixed this now. This is now corresponding to the ConvertSidToStringSid function, no longer relying on SMBJ's implementation of it.